tests/foreman/ui/test_location.py

```diff
@@ -356,6 +358,45 @@ def test_positive_add_user(self):
                     self.assertIsNotNone(element)
 
     @run_only_on('sat')
+    @skip_if_bug_open('bugzilla', 1321543)
+    @tier2
+    def test_positive_update_with_all_users(self):
+        """Create location and add user to it. Check and uncheck 'all users'
+        setting. Verify that user is assigned to location and vice versa
+        location is assigned to user
+
+        :id: 17f85968-4aa6-4e2e-82d9-b01bc17031e7
+
+        :expectedresults: Location and user entities assigned to each other
+
+        :BZ: 1321543
+
+        :CaseLevel: Integration
+        """
+        loc_name = gen_string('alpha')
+        user = entities.User().create()
+        with Session(self) as session:
+            set_context(session, org=ANY_CONTEXT['org'])
+            make_loc(session, name=loc_name)
+            self.location.update(loc_name, new_users=[user.login])
```

Comment:
Maybe i got the BZ wrong, but according to it, there was no initial location+user assignment, i.e. user was only associated via 'All Users' checkbox.
Your scenario is testing a bit different thing - it ensures user will remain associated regardless of 'All Users' checkbox state if he was associated before.

Reply:
Look closer to 1479736 BZ, but basically I was looking into exactly thing that you described. If you like I can add 1479736 id to all BZs

Reply:
That BZ describes everything, thanks. This coverage is good but BZ1321543 requires a bit different scenario. My suggestion - leave this test as is, just replace BZ to 1479736 in docstring and add similar test for BZ1321543 specifically, which won't have user assigned initially, this way we'll ensure association appears after checking off 'All Users' checkbox, like BZ1321543 states.